File: 02.PYTHON/pg-226-230/pg_227_05.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("READ TWO INTEGERS AND DISPLAY ALL THE NUMBERS ENDED IN 4 COMPRENDED BETWEEN THEM.");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("");


num1 = int(input("Write the first number: "));
num2 = int(input("Write the second number: "));

if(num1 < 0):
    num1 *= -1;

if(num2 < 0):
    num2 *= -1;
i = 1;
j = 1;
"""
while(i <= num1):
    if(i % 10 == 4):
        print(i, end=", ");
    i+=1;

print("");

while(j <= num2):
    if(j % 10 == 4):
        print(j, end=", ");
    j+=1;

"""

for i in range(i, num1+1, 1):
    if(i % 10 == 4):
        print(i, end=", ")

print("")

for j in range(j, num2+1, 1):
    if(j % 10 == 4):
        print(j, end=", ")

File: 03.JAVASCRIPT/pg-226-230/pg_227_05.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("READ TWO INTEGERS AND DISPLAY ALL THE NUMBERS ENDED IN 4 COMPRENDED BETWEEN THEM");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("");

let numInput1 = document.querySelector("#num1");
let numInput2 = document.querySelector("#num2");
let bntProof = document.querySelector(".btnProof");
let textar = document.querySelector(".textar");
let i = 1;
let j = 1;

function validateNumber()
{
    let num1 = Number(numInput1.value);
    let num2 = Number(numInput2.value);

    if(num1 < 0)
    {
        num1 *= -1;
    }
    if(num2 < 0)
    {
        num2 *= -1;
    }

    while(i <= num1)
    {
        if(i % 10 == 4)
        {
            textar.value += (i + ", ");
        }
        i++;
    }

    textar.value += "\n";

    while(j <= num2)
    {
        if(j % 10 == 4)
        {
            textar.value += (j + ", ");
        }
        j++;
    }
}

btnProof.addEventListener("click", validateNumber);

numInput1.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});
numInput2.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});